### ISSUE TITLE
Dependency issue on Py 3.9 due to click._bashcomplete

### ISSUE THREAD

**Issue by loleg** [frictionlessdata/frictionless-py#877]
Importing frictionless-py at 4.12.4 into a Python 3.9.6 environment fails due to an issue in Typer. You might want to log this upstream at https://pypi.org/project/typer/

```
remote:   File "/app/dribdat/public/api.py", line 312, in <module>
remote:     from frictionless import Package, Resource
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/frictionless/__init__.py", line 21, in <module>
remote:     from .program import program
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/frictionless/program/__init__.py", line 1, in <module>
remote:     from .api import program_api
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/frictionless/program/api.py", line 1, in <module>
remote:     from typer import Option as Opt
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/typer/__init__.py", line 29, in <module>
remote:     from .main import Typer as Typer
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/typer/main.py", line 11, in <module>
remote:     from .completion import get_completion_inspect_parameters
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/typer/completion.py", line 10, in <module>
remote:     import click._bashcomplete
remote: ModuleNotFoundError: No module named 'click._bashcomplete'
```
---

Please preserve this line to notify @roll (lead of this repository)

**Comment by loleg** [comment]
This is a known issue, linked to the version of the [Click](https://pypi.org/project/click/#history) library: https://github.com/tiangolo/typer/issues/280

Workaround: include the latest v7 release by adding `click==7.1.2` to your requirements.

**Comment by roll** [comment]
Thanks! I saw this but it doesn't look they are going to solve it.

Yea we might need to limit click version

**Comment by loleg** [comment]
Works fine for me by limiting the click version. Not sure how much of a security issue this is though.

**Comment by loleg** [comment]
Thanks @roll  !

**Comment by tiangolo** [comment]
I just released Typer `0.4.0`, which is compatible with **both** Click 7 and Click 8. :tada:

**Comment by roll** [comment]
@tiangolo Thank you!
